openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -290,6 +290,22 @@ Workspace ForwardTranslator::translateModelPrivate( model::Model & model, bool f
       pv.remove();
     }
   }
+  
+  // Remove orphan Storage
+  for (auto& storage : model.getModelObjects<ElectricalStorage>()) {
+    if (!storage.electricLoadCenterDistribution()){
+      LOG(Warn, "Electrical Storage " << storage.name().get() << " is not referenced by any ElectricLoadCenterDistribution, it will not be translated.");
+      storage.remove();
+    }
+  }
+  
+  // Remove orphan Converters
+  for (auto& converter : model.getModelObjects<ElectricLoadCenterStorageConverter>()){
```

Comment:
@jmarrec if this is a concrete type (opposed to a base class) you can use model.getConcreteModelObjects which will be faster